### ISSUE TITLE
"Default" profile should be able to be hidden or renamed

### ISSUE THREAD

**Issue by simonbcn** [kyuucr/add-link-to-qbittorrent-webui#6]
I have several profiles created: "_movies_", "_series_", "_anime_",... associated to the categories in qBittorrent with the same names. 
The name "_Default_" doesn't tell me anything. Which category do I associate it to? Which category should I define as default?
Not being able to hide it from the submenu is an option, for me, useless that only hinders the use of it. Because it's always on top of the profiles I do use and have created.
So I think this profile (_Default_) should be able to be hidden or renamed. 
In order to hide it you should check if there is an additional profile to it and, if so, enable this option.
To allow renaming the reference to it within the code must change.

**Comment by kyuucr** [comment]
Good idea, I'm okay with making the default profile hidden, but I'm not sure about renaming it since it will need a lot of changes in the profile structure.

**Comment by simonbcn** [comment]
Yes, I agree with the first option: allow making the default profile hidden. Will it be necessary to check if there are more profiles created to allow hiding it?

**Comment by kyuucr** [comment]
Yes, when there's only Default profile it is hidden by default.
